examples/image_embedding/EmbeddingModel_example_to_train_triplet_model_with_data_iterations.ipynb

```diff
@@ -1,5 +1,81 @@
 {
  "cells": [
```

Comment:
EmbeddingModel_example_to_train_triplet_model_with_data_iterations.ipynb
last p. in the overview section overflow, can you please check it out

>>> s.sessionprop.setsessopt(caslib='CASUSER', timeout=31535000)
we don't need this. the active caslib is already casuser, if you do smp where most of the time we do smp when using gpu(s).

I know this is not your part but I still wanted to document so that I can fix it later
[8] needs to be indented

[9] needs to be indented 

[11] needs to be indented

[13] needs to be indented

[15] needs to be indented



It might be a crazy idea but you think we can make the variables bold or a different color (e.g., n_samples)
"Create a batch of n_samples samples.
Train on that batch for max_epochs epochs.
Repeat 1, 2 for max_iter iterations."


again not you related, but you can comment here
>>> Note: data generation took 2.0301365852355957 (s) at iteration 0
Not sure if iteration 0 is good here. we complete a data generation iteration. what do you think?


there are so many tables after [25] and [26], can we please add some more directions and guidance here.